ooiui/static/js/views/science/HighChartsStreamingDataView.js

```diff
@@ -0,0 +1,575 @@
+                    currentView.derender();
+                    self.subviews.splice(i, 1);
+                    streamPlotRemoved= true;
+                }
+
+            }
+        });
+        return streamPlotRemoved;
+    },
+});
+
+var HighchartsStreamingDataOptionsView = Backbone.View.extend({
+    initialize: function(options) {
+        if (options && options.model){
+            this.model = options.model;
+        }
+        _.bindAll(this,'render','onPlayClick','onRemoveClick','onPauseClick');
+    },
+    events:{
+        'click #removePanel': 'onRemoveClick',
+        'click #playStream': 'onPlayClick',
+        'click #pauseStream': 'onPauseClick',
+        'click #download-plot' : 'plotDownloads',
+    },
+    onRemoveClick:function(){
+      //NOT IMPLEMENTED
+    },
+    plotDownloads: function(e) {
+        event.preventDefault();
+        var self = this;
+        self.onPauseClick();
+        if ( self.streamingDataView ){
+          var chart = self.streamingDataView.chart;
+          var fileName = chart.title.textStr + '_' + chart.subtitle.textStr;
+          chart.exportChart({type: 'image/png', filename: fileName});
+        }
+    },
+    onPlayClick:function(){
+      var self = this;
+      if (self.getVariableList().length>0){
+        this.$el.find('#playStream').prop('disabled', true);
+        if (self.streamingDataView.isRendered){
+          self.streamingDataView.chart.showLoading();
+          self.streamingDataView.chart.isLoading=true;
+          self.streamingDataView.updateVariable(self.getVariableList());
+          self.streamingDataView.resume();
+        }else{
+          self.streamingDataView.updateVariable(self.getVariableList());
+          self.streamingDataView.render();
+        }
+        this.$el.find('#pauseStream').prop('disabled', false);
+        this.$el.find('#paramSelection').attr('disabled',true);
+        this.$el.find('.selectpicker').selectpicker('refresh');
+      }
+    },
+    onPauseClick:function(){
+      var self = this
+      this.$el.find('#pauseStream').prop('disabled', true);
+      self.streamingDataView.abort();
+      this.$el.find('#playStream').prop('disabled', false);
+      this.$el.find('#paramSelection').attr('disabled',false);
+      this.$el.find('.selectpicker').selectpicker('refresh');
+    },
+    template: JST['ooiui/static/js/partials/HighChartsStreamingDataOptionsView.html'],
+    initialRender: function() {
+        this.$el.html('<i class="fa fa-spinner fa-spin" style="margin-top:80px;margin-left:50%;font-size:90px;"> </i>');
+        return this;
+    },
+    getVariableList:function(){
+      var self = this;
+      var selectedItem =  self.$el.find("#paramSelection option:selected");
+      var selected = [];
+      $(selectedItem).each(function(index){
+        selected.push({'variable':$(this).data('params'),'prettyname':$(this).text()});
+      });
+      return selected;
+    },
+    render: function() {
```

Comment:
@birdage this could be tidier, but it looks like you wanted to reuse code from SVGPlotting which is cool.  Not a show stopper!